README.md

```diff
@@ -282,9 +284,10 @@ Table of Contents
 - [**KeePassDX**](https://github.com/Kunzisoft/KeePassDX) <sup>**[[F-Droid](https://f-droid.org/app/com.kunzisoft.keepass.libre)]**</sup>
 - [**Aegis**](https://getaegis.app/) <sup>**[[F-Droid](https://f-droid.org/app/com.beemdevelopment.aegis)]**</sup>
 - [**Bitwarden Mobile**](https://github.com/bitwarden/mobile)
-
+- [**Passwd**](https://github.com/passwdapp/passwd)
 
 ## • Podcast & Audio Book Player
+- [**AntennaPod**](https://github.com/AntennaPod/AntennaPod)
```

Comment:
:+1: It's available on F-Droid too: https://f-droid.org/app/de.danoeh.antennapod

Reply:
✌